development/idp-local/config/config.php

```diff
@@ -63,6 +83,11 @@
      */
     'tempdir' => '/tmp/simplesaml',
 
+    /*
+     * Name of this IdP to display to the user
+     */
+    'idp_name' => $IDP_NAME,
```

Comment:
Not that it affects this PR, but here's an example of where the expectation is that `$IDP_NAME` contains a user-friendly version (rather than something like a URL-friendly version).